components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.common/src/main/java/org/wso2/carbon/identity/api/server/branding/preference/management/common/BrandingPreferenceManagementConstants.java

```diff
@@ -54,6 +58,14 @@ public enum ErrorMessage {
         ERROR_CODE_NOT_ALLOWED_BRANDING_PREFERENCE_CONFIGURATIONS("60004",
                 "Not allowed branding preference configurations.",
                 "Requested branding preference configuration: %s is not allowed for the organization."),
+        ERROR_CODE_INVALID_CUSTOM_TEXT_PREFERENCE("60005",
+                "Invalid Custom Text Preference configurations.",
```

Comment:
Any reason to use `Title Case` for `Invalid Custom Text Preference`  if not shall we improve it?
```suggestion
                "Invalid custom text preference configurations.",
```

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/495/commits/f15e2dac78ced6aee72e580d1fad018ea99574c3